mkosi.conf

```diff
@@ -5,9 +5,9 @@
 MinimumVersion=23~devel
 
 [Output]
-@OutputDirectory=mkosi.output
-@BuildDirectory=mkosi.builddir
-@CacheDirectory=mkosi.cache
+@OutputDirectory=build/mkosi.output
```

Comment:
It's common at least for me to have multiple build directories in parallel with different configs, can this be derived at runtime from the meson config instead of being hardcoded?

Reply:
@bluca There is no "meson" config, why do you think it's `ninja -C build` instead of `ninja? ;)

The only way we could make this work is we add a dropin to each build directory and force everyone to run `mkosi -C build ...` instead of just `mkosi`.

Reply:
I can make this work, but it requires us to get rid of mkosi.images, which we can do after this and some other work land.

Reply:
is there any specific reason to move it in the first place?

Reply:
These are build outputs and build outputs should be in the build directory. We're currently littering the source directory with build artifacts which we shouldn't do.